source/common/ext_authz/ext_authz_impl.cc

```diff
@@ -0,0 +1,250 @@
+#include "common/ext_authz/ext_authz_impl.h"
```

Comment:
Why was any of this called `ext_authz` in the first place? I get the `ext` bit, but what is the `z` in `authz`?

Reply:
z: Authori**Z**ation; it is to differentiate from auth (authe**N**tication).

Is the naming of files ok? I've changed it twice already (before the pr) :-)
Would like to do it just once more, if needed.

Reply:
Yep, this is fine, thanks for the explanation.